docs/install.rst

```diff
@@ -0,0 +1,35 @@
+Installation
+============
+
+Installation of Cascade
+-----------------------
+Cascade interacts with Dismod-AT underneath. Cascade runs Dismod-AT within
+the IHME infrastructure. Clone it from
+`Cascade on Github <https://github.com/ihmeuw/cascade>`_. Then
+use ``python setup.py install`` to install it into the current
+Python environment or virtual environment.
+
+
+
+Installation of Dismod-AT
+-------------------------
+This describes how to run Dismod-AT on your local computer, not on the
+cluster. It uses Docker, which you have to install separately.
+
+Download and run from the IHME Docker registry. This requires
+VPN access. Maybe you can pull images without the login?::
+
+    docker login reg.ihme.washington.edu
```

Comment:
When I try this, (and I am connected to VPN - there's a big red dot in the upper right on my monitor), I am getting a 'permission denied' error: 

 nhawkins@nh-tp:~$ docker login reg.ihme.washington.edu

Warning: failed to get default registry endpoint from daemon (Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get http://%2Fvar%2Frun%2Fdocker.sock/v1.27/info: dial unix /var/run/docker.sock: connect: permission denied). Using system default: https://index.docker.io/v1/
Username: nhawkins
Password: 
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post http://%2Fvar%2Frun%2Fdocker.sock/v1.27/auth: dial unix /var/run/docker.sock: connect: permission denied

Reply:
That's interesting. I don't know why. You're an admin on that registry.